evap/staff/views.py

```diff
@@ -671,6 +674,8 @@ def semester_make_active(request):
 @require_POST
 @manager_required
 def semester_delete(request):
+    # TODO(rebeling): Do we expect reward point granting here? Do we want to notify? I don't see why it couldn't happen
```

Comment:
@janno42 I didn't find anything in the code that prohibits deleting semesters with active rewards. I think it's possible that we also get a reward point granting here? Do we want that? (If yes, I guess we want a matching `messages.info`)